sklearn/inspection/tests/test_plot_partial_dependence.py

```diff
@@ -380,6 +384,10 @@ def test_plot_partial_dependence_dataframe(pyplot, clf_boston, boston):
         clf_boston, df, ['TAX', 'AGE'], grid_resolution=grid_resolution,
         feature_names=df.columns.tolist()
     )
+    plot_individual_conditional_expectation(
+        clf_boston, df, ['TAX', 'AGE'], grid_resolution=grid_resolution,
```

Comment:
We should switch from Boston to another dataset from now on. Since we are touching the test, feel free to take another one. I think that you can take another regression toy dataset for the purpose of these unit tests. We are only checking for operational behaviour and not results.